packages/fluent/scss/tabstrip/_layout.scss

```diff
@@ -160,15 +162,156 @@
     // Scrolling
     .k-tabstrip-scrollable {
         > .k-tabstrip-items-wrapper {
-
             > .k-tabstrip-items {
                 flex-wrap: nowrap;
                 white-space: nowrap;
                 overflow: hidden;
+
+                &.k-tabstrip-items-scroll {
+                    scrollbar-width: none;
+
+                    &::-webkit-scrollbar {
+                        display: none;
+                    }
+                }
+            }
+
+            &.k-tabstrip-top,
+            &.k-tabstrip-bottom {
+                .k-tabstrip-items.k-tabstrip-items-scroll {
+                    overflow-x: auto;
+                }
+            }
+
+            &.k-tabstrip-left,
+            &.k-tabstrip-right {
+                .k-tabstrip-items.k-tabstrip-items-scroll {
+                    overflow-y: auto;
+                }
             }
         }
+
```

Comment:
Shall we leave these styles since they are inside a comment and I assume we don't need them